Makefile

```diff
@@ -97,6 +97,11 @@ test-cleanup-azure-resources:
 		az group delete --name $$rgname --no-wait --yes; \
 	done
 
+	for rgname in `az group list --query "[*].[name]" -o table | grep 'rg-prime$$' `; do \
+    	echo "$$rgname will be deleted"; \
+    	az group delete --name $$rgname --no-wait --yes; \
+    done
```

Comment:
need an explanation here since this change appears to be out of scope

Reply:
I can break it out into it's own PR.

Basically, running the tests can leak resources (at least locally?) because the tests are suffixed with rg-prime. Let me know if you want me to move this to its own PR.